pkg/webhooks/validator/validator_test.go

```diff
@@ -1186,6 +1187,7 @@ var _ = Describe("webhooks validator", func() {
 		})
 
 		Context("validate moved FG on update", func() {
+			//nolint:staticcheck
```

Comment:
I would add a space at the start of the line like other places in the pr

Reply:
This is a special comment, for the linter. it must be w/o a space